server/grapevine-api/middlewares/validateRequest.coffee

```diff
@@ -1,33 +1,26 @@
 jwt = require 'jwt-simple'
-# retrieveUser = require('../routes/users').retrieveUser
+retrieveUser = require('../routes/auth').retrieveUser
 
```

Comment:
Is there a better endpoint name than `auth` for this? What is the resource we are trying to use?